packages/front-generator/src/generators/common/BaseTsAppGenerator.ts

```diff
@@ -0,0 +1,63 @@
+import {appGenerationOptions, CommonGenerationOptions, OptionsConfig} from "../../common/cli-options";
+import {BaseGenerator} from "../../common/base-generator";
+import {AppTemplateModel} from "../react-typescript/app";
+import {StudioTemplateProperty} from "../../common/studio/studio-model";
+import {AppAnswers, appGeneratorParams} from "../react-typescript/app/params";
```

Comment:
I think it's better to move things like `AppTemplateModel`, `AppAnswers`, `appGeneratorParams` from the `react-typescript/app` generator to the `BaseTsAppGenerator`.